android/keystores/debug.keystore.properties

```diff
@@ -1,4 +0,0 @@
-key.store=debug.keystore
```

Comment:
any specific reason to remove this file?

Reply:
Idea about the debug keystore is that the developer should have his/her own debug.keystore file. It is OK to use a common file but not a good standard.